src/avatar-group.scss

```diff
@@ -0,0 +1,102 @@
+@import "./mixins";
+@import "./avatar";
+
+/*!
+.fd-avatar-group+((--individual|--group), (--xxs|--xs|--s|--m|--l|--xl|--xxl)
+*/
+$block: #{$fd-namespace}-avatar-group;
+
+.#{$block} {
+  $fd-avatar-group-item-spacing: (
+    "individual": (
+      "xs": ("margin-right": 0.0625rem),
+      "s": ("margin-right": 0.125rem),
+      "m": ("margin-right": 0.125rem),
+      "l": ("margin-right": 0.125rem),
+      "xl": ("margin-right": 0.25rem)
+    ),
+    "group": (
+      "xs": ("margin-right": -0.75rem),
+      "s": ("margin-right": -1.25rem),
+      "m": ("margin-right": -1.625rem),
+      "l": ("margin-right": -2rem),
+      "xl": ("margin-right": -2.75rem)
+    )
+  );
+  $fd-const-ratio: 1rem;
+  $fd-avatar-group-more-button-sizes: (
+    "xs": ("ratio": ($fd-const-ratio * 2), "font-size": 1rem),
+    "s": ("ratio": ($fd-const-ratio * 3), "font-size": 1.125rem),
+    "m": ("ratio": ($fd-const-ratio * 4), "font-size": 1.5rem),
+    "l": ("ratio": ($fd-const-ratio * 5), "font-size": 2.25rem),
+    "xl": ("ratio": ($fd-const-ratio * 7), "font-size": 3rem)
+  );
+
+  @include fd-reset();
+
+  &--individual {
+    .#{$block}__item {
+      @include fd-fiori-focus();
+      @include action-cursor();
+    }
+  }
+
+  &--group {
+    @include fd-fiori-focus();
+    @include action-cursor();
+
+    .#{$block}__more-button {
+      pointer-events: none;
+    }
+  }
+
+  &__item {
+    @include fd-reset();
+
+    display: inline-block;
+  }
+
+  &__more-button {
+    @include fd-reset();
+
+    .fd-button__text {
+      font-size: inherit;
+
+      bdi {
```

Comment:
what is this?

Reply:
@InnaAtanasova 

> The `<bdi>` element is used to wrap a span of text and instructs the bidirectional algorithm to treat this text in isolation from its surroundings. This works in two ways:
> 
> - The directionality of text embedded in `<bdi>` does not influence the directionality of the surrounding text.
> - The directionality of text embedded in `<bdi>` is not influenced by the directionality of the surrounding text.

 In our case to prevent changes from `+9` to `9+`

Reply:
Could you add some class with `fd-reset` and inherit `font-size`/`color`/`weight`? Or even use `bdi` with `fd-button__text` , so you can get rid of his selector